src/main/kotlin/no/nav/medlemskap/regler/funksjoner/StatsborgerskapFunksjoner.kt

```diff
@@ -17,6 +17,28 @@ object StatsborgerskapFunksjoner {
             Periode(it.fom, it.tom).interval().contains(lagInstantStartOfDay(dato))
         }.map { it.landkode }
 
+    infix fun List<Statsborgerskap>.harEndretSisteÅret(kontrollPeriode: Kontrollperiode): List<Statsborgerskap> =
+        this.filter {
+            erStatsborgerskapetInnenforPerioden(it, kontrollPeriode)
+        }
+
+    private fun erStatsborgerskapetInnenforPerioden(it: Statsborgerskap, kontrollPeriode: Kontrollperiode): Boolean =
```

Comment:
Mulig denne metoden kan skrives bedre. Tenker i retning av at man kan erstatte NULL med min/max, avhengig av om det er fom eller tom. Kan gjerne bruke elvis operator til å sette verdi til min eller max dersom fom/tom er null. Deretter holder det å gjøre en overlapp-sjekk mellom de to periodene.